src/main/docs/guide/introduction.adoc

```diff
@@ -1 +1,16 @@
 The `micronaut-cassandra` module includes support for integrating Micronaut services with Cassandra.
+
+=== Release History
```

Comment:
This is a good fit and you're right, didn't necessarily need to be in its own `changelog.adoc` file